dnf-behave-tests/dnf/security-security.feature

```diff
@@ -11,9 +11,8 @@ Background: Use repository with advisories
         | install       | security_B-0:1.0-1.x86_64 |
 
 
-# @dnf5
-# TODO(nsella) Unknown argument "check-update" for command "microdnf"
-Scenario: Security check-update when there are no such updates
+@dnf5
```

Comment:
If I see it correctly, now we can set a global `@dnf5` tag for this whole feature file.

Reply:
Done, thanks.